Client/Frontend/Widgets/SimpleHighlightCell.swift

```diff
@@ -53,40 +35,44 @@ class SimpleHighlightCell: UITableViewCell {
         return titleLabel
     }()
 
-    lazy var descriptionLabel: UILabel = {
-        let titleLabel = UILabel()
-        titleLabel.font = DynamicFontHelper.defaultHelper.DeviceFontDescriptionActivityStream
-        titleLabel.textColor = SimpleHighlightCellUX.DescriptionLabelColor
-        titleLabel.textAlignment = .Left
-        titleLabel.numberOfLines = 1
-        return titleLabel
+    private lazy var descriptionLabel: UILabel = {
```

Comment:
I like to make sure everything is private so you know when a property might be edited outside of its view.